_includes/getting-started.html

```diff
@@ -1,22 +1,111 @@
-<div class="anchor"><a name="getting-started"></a><h2>Run Meshery</h2></div>
-<div class="container flex">
+<html>
+  <head>
+    
```

Comment:
Given that this is a partial html snippet that will be shown in context of a larger page, we'll want to remove the `<html>` and <head> tags.